eng/pipelines/runtime-richnav.yml

```diff
@@ -0,0 +1,43 @@
+trigger:
+  branches:
+    include:
+    - master
+    - dev/infrastructure
```

Comment:
I think `dev/infrastructure` is not a very common branch to navigate on, so might make sense to not include it.